### ISSUE TITLE
yarn install broken with the M1 arm64 chip

### ISSUE THREAD

**Issue by jsappme** [gridsome/gridsome#1411]
When I run the yarn command to install the node modules, I get this error message:

```
% yarn
yarn install v1.22.5
[1/4] 🔍  Resolving packages...
warning gridsome > @hapi/joi@15.1.1: Switch to 'npm install joi'
warning gridsome > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning gridsome > chokidar > fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
warning gridsome > probe-image-size > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning gridsome > webpack > watchpack > chokidar > fsevents@2.1.3: Please update to v 2.2.x
warning gridsome > webpack > watchpack > watchpack-chokidar2 > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > @fortawesome/vue-fontawesome@0.1.10" has unmet peer dependency "vue@~2".
warning " > bootstrap@4.5.3" has unmet peer dependency "jquery@1.9.1 - 3".
warning " > bootstrap@4.5.3" has unmet peer dependency "popper.js@^1.16.1".
warning "bootstrap-vue > portal-vue@2.1.7" has unmet peer dependency "vue@^2.5.18".
warning " > gridsome-plugin-pwa@0.0.18" has unmet peer dependency "sharp@0.25.x".
warning " > vue-svg-loader@0.15.0" has unmet peer dependency "vue-template-compiler@^2.0.0".
warning "vue-svg-loader > svg-to-vue@0.6.0" has unmet peer dependency "vue-template-compiler@^2.0.0".
warning " > sass-loader@8.0.0" has unmet peer dependency "webpack@^4.36.0".
[4/4] 🔨  Building fresh packages...
[16/16] ⢀ node-sass
[-/16] ⢀ waiting...
[-/16] ⢀ waiting...
[12/16] ⢀ sharp
error /Users/hervefulchiron/Documents/GitHub/spt-front/node_modules/sharp: Command failed.
Exit code: 1
Command: (node install/libvips && node install/dll-copy && prebuild-install --runtime=napi) || (node-gyp rebuild && node install/dll-copy)
Arguments: 
Directory: /Users/hervefulchiron/Documents/GitHub/spt-front/node_modules/sharp
Output:
info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.9.1/libvips-8.9.1-darwin-arm64v8.tar.gz
ERR! sharp Prebuilt libvips 8.9.1 binaries are not yet available for darwin-arm64v8
info sharp Attempting to build from source via node-gyp but this may fail due to the above error
info sharp Please see https://sharp.pixelplumbing.com/install for required dependencies
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@15.5.0 | darwin | arm64
gyp info find Python using Python version 3.8.2 found at "/Library/Developer/CommandLineTools/usr/bin/python3"
gyp info spawn /Library/Developer/CommandLineTools/usr/bin/python3
```

Node version v15.5.0

**Comment by noxify** [comment]
Based on the error messages, I think this is related to https://github.com/lovell/sharp/issues/2460

**Comment by jsappme** [comment]
here is the solution

**Comment by sambigelow** [comment]
I'm not necessarily trying to reopen this - but I can see in the bottom of the issue mentioned above that the better fix is to install sharp 0.28.1. Which it looks like y'all have upped in the [gridsome package.json](https://github.com/gridsome/gridsome/blob/84fd8aafdd16105d1b9dacaaf9a0f1881651d384/gridsome/package.json#L84). But I don't think it's in the release yet. It looks like it was merged 2 months ago. Is it going to be released soon?

**Comment by jubarcelos** [comment]
I'm in my adm user, and this message come in every time I try install Yarn on my OS M1.
Is there anyone who has passed for this situation here and can give me some help? Tks!

```
~ % npm install --global yarn
npm WARN checkPermissions Missing write access to /usr/local/lib/node_modules
npm ERR! code EACCES
npm ERR! syscall access
npm ERR! path /usr/local/lib/node_modules
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, access '/usr/local/lib/node_modules'
npm ERR!  [Error: EACCES: permission denied, access '/usr/local/lib/node_modules'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib/node_modules'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/julia/.npm/_logs/2021-10-17T16_25_56_470Z-debug.log
 ~ % 

```
